docs/process_joined_data.js

```diff
@@ -52,24 +52,25 @@ function colorAndRate(
  * @param {number} scalingFactor multiplies the raw score, it can be
  *     adjusted to make sure that the values span the desired range of ~0 to
  *     ~100.
- * @param {number} povertyThreshold between 0 and 1 representing what
- *     fraction of a population must be SNAP eligible to be considered.
- * @param {number} damageThreshold a number between 0 and 1 representing what
- *     fraction of a block group's building must be damaged to be considered.
- * @param {number} povertyWeight float between 0 and 1 that describes what
- *     percentage of the score should be based on poverty (this is also a proxy
- *     for damageWeight which is 1-this value).
+ * @param {Promise<Array<number>>} initialTogglesValuesPromise promise
+ * that returns the poverty and damage thresholds and the poverty weight (from
+ * which the damage weight is derived).
  * @return {Promise}
  */
 function processJoinedData(
-    dataPromise, scalingFactor, povertyThreshold, damageThreshold,
-    povertyWeight) {
-  return dataPromise.then((featureCollection) => {
-    for (const feature of featureCollection.features) {
-      colorAndRate(
-          feature, scalingFactor, povertyThreshold, damageThreshold,
-          povertyWeight);
-    }
-    return featureCollection.features;
-  });
+    dataPromise, scalingFactor, initialTogglesValuesPromise) {
+  return Promise.all([dataPromise, initialTogglesValuesPromise])
+      .then((results) => {
+        const featureCollection = results[0];
```

Comment:
can destructure in one line, like `const [featureCollection, [povertyThreshold, ...]] = results;` if you take my suggestion below, just change inner square brackets to curlies.